src/stores/my-account.ts

```diff
@@ -5,20 +5,42 @@ import {
   loginWithSecretKey,
   Signer,
 } from '@/utils/account'
+import { getWeb3AuthClientId } from '@/utils/env/client'
+import { generateRandomName } from '@/utils/random-name'
 import { LocalStorage } from '@/utils/storage'
+import { SubstrateRPC } from '@/utils/substrate-rpc'
+import * as bottts from '@dicebear/bottts'
+import { createAvatar } from '@dicebear/core'
+import { CHAIN_NAMESPACES, SafeEventEmitterProvider } from '@web3auth/base'
+import { Web3Auth } from '@web3auth/modal'
 import { useAnalytics } from './analytics'
 import { create } from './utils'
 
+const clientId = getWeb3AuthClientId()
+
+type AuthenticatedUser = {
+  name: string
+  encodedSecretKey: string
+  address: string
```

Comment:
address also doubles, I think this AuthenticatedUser just needs some additional user data that web3auth provides, like name email username, profilepic
Others are also there in anon login right? so let's just make the AuthenticatedUser like a special object that only web3auth login provides.

I think the name is also better to be just `User`

Reply:
Honestly, I think we can kind of do without it because we don't really use the user's info as returned from the web3Auth. It would have been more useful if we were going to be using that.
For now, I intended to make it take only the name, email and original profile pic as returned from Web3Auth but since we don't use it, this might as well not be in the store